src/Components/SmartComponents/Systems/Systems.js

```diff
@@ -90,27 +86,22 @@ const SystemsHook = ({ intl, location }) => {
         loadInventory();
     }, []);
 
+    useEffect(() => () => {
+        dispatch(clearSystemStore());
```

Comment:
you have another `useEffect` that runs on mount.   What's the reason for having it in a  different useEffect? 


Also, what's the case of resetting the store on mount?

```suggestion
        useEffect(() => {
        loadInventory();
        // Cleanup for effect from first render
        return () => {
         dispatch(clearSystemStore());
         }
    }, []);
```

Reply:
Because I thought it would be better if I separate onMount and onUnmount into separate hooks.  I might have over thought. It makes sense. I will get rid of the next hook.